tiledb/api/src/array/schema/strategy.rs

```diff
@@ -103,9 +108,47 @@ pub fn prop_schema_for_domain(
                 offsets_filters,
                 nullity_filters,
             )| {
+                /*
+                 * Update the set of dimension/attribute names to be unique.
+                 * This probably ought to be threaded into the domain/attribute strategies
+                 * so that they have unique names in all scenarios, but this is way
+                 * easier as long as we only care about the Schema in the end.
+                 */
```

Comment:
Do you have a rough idea of what percentage of schemas had non-unique field names? Remember that one of things we can use proptest for is asserting when an error should happen. I.e., instead of unique-ifying names we could turn this into a "if we have duplicates, assert that the schema fails to build". Though if its a fairly high percentage of cases then this is also reasonable as well.

Reply:
I think it was pretty high because our name generator does allow zero and one-character names.